examples/account/private_tangle.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! A basic example that generates, publishes and deletes a DID Document
+//! to and from a private tangle.
+//! It can be run together with a local hornet node.
+//! Refer to https://github.com/iotaledger/one-click-tangle/tree/chrysalis/hornet-private-net
+//! for setup instructions.
+//!
+//! cargo run --example account_private_tangle
+
+use identity::account::Account;
+use identity::account::IdentityCreate;
+use identity::account::IdentitySnapshot;
+use identity::account::Result;
+use identity::iota::IotaDID;
+use identity::iota::IotaDocument;
+use identity::iota::Network;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+  pretty_env_logger::init();
+
+  let network_name = "private-tangle";
+
+  // Unwrap is fine since we provided a non-empty string.
+  let network = Network::from_name(network_name).unwrap();
+
+  // Create a new Account with the default configuration
+  let account: Account = Account::builder()
+    // Configure a client for the private network.
+    // Also set the URL that points to the REST API
+    // of the locally running hornet node.
+    .client(network, |builder| {
+      // unwrap is safe, we provided a valid node URL
+      builder.node("http://127.0.0.1:14265/").unwrap()
```

Comment:
Can we just use `?` instead of `unwrap()` to return the error for brevity?

Reply:
I changed the other occurrences of `unwrap` to use `?`. Since this one is inside a closure, we can't use `?`.